### PR TITLE
update substrate reference

### PR DESCRIPTION
**Pull request by rphmeier**

### CODE REVIEW

File: runtime/src/lib.rs [hunk 1]
```diff
@@ -188,6 +188,22 @@ impl session::Trait for Runtime {
 	type Event = Event;
 }
 
+/// Converter for currencies to votes.
+pub struct CurrencyToVoteHandler;
```
Comment (rphmeier) [inline runtime/src/lib.rs]: cc @Kianenigma copied from substrate-node. is it ok?

File: runtime/src/parachains.rs [hunk 2]
```diff
@@ -72,11 +72,11 @@ decl_storage! {
 
 			let only_ids: Vec<_> = p.iter().map(|&(ref id, _, _)| id).cloned().collect();
 
-			storage.insert(Self::hash(<Parachains<T>>::key()).to_vec(), only_ids.encode());
+			storage.insert(<Parachains<T>>::key().to_vec(), only_ids.encode());
 
 			for (id, code, genesis) in p {
-				let code_key = Self::hash(&<Code<T>>::key_for(&id)).to_vec();
-				let head_key = Self::hash(&<Heads<T>>::key_for(&id)).to_vec();
+				let code_key = <Code<T>>::key_for(&id);
```
Comment (rphmeier) [inline runtime/src/parachains.rs]: cc @thiolliere This change isn't right, but I don't know how the API has changed

Comment (thiolliere) [reply runtime/src/parachains.rs]: https://github.com/paritytech/polkadot/pull/245

Comment (thiolliere) [reply runtime/src/parachains.rs]: otherwise we should use twox128 for StorageValue and blake2_256 for maps

File: runtime/src/lib.rs (hunk 1 above)
Comment (Kianenigma) [reply runtime/src/lib.rs]: Yup. This is also the new version. Worth keeping an eye on Staking in the next test-net runtime upgrade.

Comment (rphmeier) [reply runtime/src/lib.rs]: The PR that would affect the next runtime upgrade is https://github.com/paritytech/polkadot/pull/247 (against poc-4 branch)

**Pull request merged**